pkg/actors/actors.go

```diff
@@ -63,6 +64,11 @@ const (
 
 	errStateStoreNotFound      = "actors: state store does not exist or incorrectly configured"
 	errStateStoreNotConfigured = `actors: state store does not exist or incorrectly configured. Have you set the property '{"name": "actorStateStore", "value": "true"}' in your state store component file?`
+
+	envKeyHealthFailureThreshold = "DAPR_EXPERIMENTAL_ACTOR_HEALTH_FAILURE_THRESHOLD"
```

Comment:
Have we agreed to the "DAPR_EXPERIMENTAL" as the official way to introduce env variables?

I don't have a problem with it. Just checking we had done the vetting on this.

Reply:
No we haven't, and I'd definitely like feedback for it. It _seems_ like the right thing to do, but there may already exist a mechanism in Dapr I'm not aware of.

Ideally, we would merge the actor and app health checks into the same routine which would drop the need for this at all.

The main reason for this to exist is for integration tests not to take an unreasonable amount of time, though real users could see benefit from it in theory.